docs/decisions/0008-frontend-plugin-fallback.rst

```diff
@@ -0,0 +1,38 @@
+How to Handle Frontend iFrame Plugin Fallback 
+======================================================================================
+
+Status
+------
+
+Draft
+
+Context
+-------
+The Frontend Plugin Framework's iFrame class implementation will live in Frontend Platform,
+but the question of how we want to handle the case where the iFrame fails to render needs to
+handled. 
+
+Given that this is a new feature that is being made available to the frontend, we 
+can't anticipate all of the ways that the plugin will be used for a given repo, nor do we 
+have any intention to limit the scope of how it can be used. Because we don't all of the use 
+cases for the plugin, we also don't know the desired fallback method that is desired for any 
+given plugin.
+
+Decision
+--------
+For the above reasons, we will have failing iFrame plugins return an H1 tag that 
```

Comment:
[curious] Is there an opportunity to use an `ErrorBoundary` in some way here? Either the one defined/used by `@edx/frontend-platform` or a custom one for this use case.

Reply:
Ah! You know, I should update this to name that there is a `PluginErrorBoundary` component that is part of the PR I'm working on for this. 

In short, the `PluginErrorBoundary` will capture any errors that occur within, and whatever decision is made here on how to best reflect an error will be handled by the error boundary.